provisioning/playbooks/roles/grpc/tasks/main.yml

```diff
@@ -0,0 +1,48 @@
+
+- name: install grpc base dependencies
+  apt: name={{item}}
+  become: yes
+  become_user: root
+  with_items:
+      - python-dev
+      - python-pip
+      - libtool
+      - autoconf
+
+- name: update pip
```

Comment:
you need to install it first!
Why not make the pip install + upgrade a separate role and reuse that (also in the client provisioning script)